code/modules/mob/living/simple_animal/abnormality/waw/judgement_bird.dm

```diff
@@ -97,7 +97,7 @@
 	playsound(get_turf(src), 'sound/abnormalities/judgementbird/pre_ability.ogg', 50, 0, 2)
 	SLEEP_CHECK_DEATH(2 SECONDS)
 	playsound(get_turf(src), 'sound/abnormalities/judgementbird/ability.ogg', 75, 0, 7)
-	for(var/mob/living/L in urange(judgement_range, src))
+	for(var/mob/living/L in livinginrange(judgement_range, src))
```

Comment:
Agonizing and significantly more laggy, This won't do for now; Please try to find another way. There is a ways to make damage pierce rhinos I just forgot what it was